src/execsnoop/bpf.c

```diff
@@ -0,0 +1,135 @@
+#include <uapi/linux/ptrace.h>
```

Comment:
Can we include a link to the original like we have in the other ported tools?

Reply:
Added link (and copyright information) to the original in-line bpf code from https://github.com/iovisor/bcc/blob/master/tools/execsnoop.py